pkg/apis/options/providers.go

```diff
@@ -62,6 +62,8 @@ type Provider struct {
 
 	// LoginURL is the authentication endpoint
 	LoginURL string `json:"loginURL,omitempty"`
+	// LoginURLParameters defines the parameters that can be passed from the start URL to the IdP login URL
+	LoginURLParameters map[string]LoginURLParameter `json:"loginURLParameters,omitempty"`
```

Comment:
Normally we use a list and have a name within the list rather than having a map, I don't think we have any maps in the config at the moment.

This has come from me being a K8s developer primarily where maps are generally discouraged, I can't think why right now though, will think if there are disadvantages that might cause issue here